src/tools/compiletest/src/main.rs

```diff
@@ -211,11 +213,24 @@ pub fn parse_config(args: Vec<String>) -> Config {
         // Avoid spawning an external command when we know tidy won't be used.
         false
     };
+
+    let target_cfg = if cfg!(test) {
+        HashMap::new()
+    } else {
```

Comment:
I'm not sure this is great, but I don't know how else to get around the fact that a rustc_path seems not to be provided for test runs. There could be something obvious I'm missing though. I'm pretty tired.

Reply:
Hm, it seems likely that should be fixable- maybe a separate PR is best, but it seems like we can pass mostly the same args during tests via an env variable or something...

Can you leave a FIXME comment here perhaps?

Reply:
Sure!